hazelcast-jet-core/src/main/resources/client-protocol-definition/Jet.yaml

```diff
@@ -81,13 +81,23 @@ methods:
     request:
       retryable: true
       partitionIdentifier: -1
-      params: []
+      params:
+        - name: onlyName
+          type: String
+          nullable: true
+          since: 2.3
+          doc: ''
+        - name: onlyJobId
+          type: long
+          nullable: false
+          since: 2.3
+          doc: ''
     response:
       params:
         - name: response
-          type: List_Long
+          type: Data
```

Comment:
@viliam-durina 
Should we care about backward compatibility for Jet clients that uses `getJobIds`?
An already released jet client will not be able to work with the new version of the protocol. It will not be able to parse the response.

Reply:
Jet doesn't have any client b-w compatibility. We use Data elsewhere too.